gigasecond/gigasecond_test.spec.js

```diff
@@ -0,0 +1,23 @@
+import Gigasecond from './gigasecond';
+
+describe('Gigasecond', () => {
+
+  it('tells the gigasecond birth of a living person', () => {
```

Comment:
I'm skeptical of this one. Being born in 1977 doesn't guarantee that they're alive now.